services/apollo/Dockerfile

```diff
@@ -1,16 +1,18 @@
-ARG NODE_VERSION=${NODE_VERSION:-14.15.1}
-FROM node:${NODE_VERSION}-buster-slim
-
+# Nodejs version, default to 14
+ARG NODE_VERSION=${NODE_VERSION:-14}
 # Prefect Version, default to MASTER
 ARG PREFECT_VERSION
-ENV PREFECT_VERSION=${PREFECT_VERSION:-master}
-
 # Prefect Server Version, default to MASTER
 ARG PREFECT_SERVER_VERSION
-ENV PREFECT_SERVER_VERSION=${PREFECT_SERVER_VERSION:-master}
-
+# Prefect Apollo Release timestamp
 ARG RELEASE_TIMESTAMP
+
+FROM node:${NODE_VERSION}-buster-slim
+
+ENV PREFECT_VERSION=${PREFECT_VERSION:-master}
```

Comment:
these ENVs probably belong below, either above the ENTRYPOINT or below the COPY . .